internal/config/config.go

```diff
@@ -28,6 +28,7 @@ type NeutronQueryRelayerConfig struct {
 	QueriesTaskQueueCapacity    int                      `split_words:"true" default:"10000"`
 	PrometheusPort              uint16                   `split_words:"true" default:"9999"`
 	InitialTxSearchOffset       uint64                   `split_words:"true" default:"0"`
+	IgnoreErrorsRegex           string                   `split_words:"true" default:"(execute wasm contract failed)"`
```

Comment:
1) parentheses are not needed in this case. without a `|` inside, they work as a group matcher. this could confuse users and it probably affects rx performance;
2) why you decided to make it the way users define errors to ignore, not errors to crash? say, I want my app to only crash on `execute wasm contract failed`. how do I do it? from the other hand, this approach will make the opposite scenario barely achievable. maybe we should let users define both lists — what to ignore and when to crash?

Reply:
1. so suddenly now it has `|` :)
2. I just implemented logic that we agreed on. Anyway IMO letting users configure what to ignore and when to crash is really misleading